static/_redirects

```diff
@@ -681,14 +680,100 @@
+/developer/build-on-minipay/deeplinks /build/build-on-minipay/deeplinks
+/protocol/oracle/index /developer/oracles/
+/protocol/oracle/run /developer/oracles/run
+/protocol/oracle/band-protocol /developer/oracles/band-protocol
+/protocol/oracle/run /developer/oracles/run
+/protocol/oracle/chainlink-oracles /developer/oracles/chainlink-oracles
+/protocol/oracle/redstone /developer/oracles/redstone
+/protocol/oracle/supra /developer/oracles/supra
+/protocol/bridges /developer/bridges/
+/protocol/cross-chain-messaging /developer/bridges/cross-chain-messaging
+/general /what-is-celo
+/general/ecosystem/overview /what-is-celo/joining-celo/index
+/general/ecosystem/builders /what-is-celo/joining-celo/builders
+/general/ecosystem/contributors /what-is-celo/joining-celo/contributors/overview
+/general/ecosystem/overview /what-is-celo/joining-celo/index
+/general/architecture /what-is-celo/about-celo-l1/l1-architecture
+/protocol/index /what-is-celo/about-celo-l1/protocol
+/protocol/governance /what-is-celo/about-celo-l1/protocol/governance
+/protocol/stability /what-is-celo/about-celo-l1/protocol/stability
+/protocol/proof-of-stake /what-is-celo/about-celo-l1/protocol/pos
+/protocol/bridging /what-is-celo/about-celo-l1/protocol/bridge
+/protocol/pos/index /what-is-celo/about-celo-l1/protocol/pos/index
+/protocol/pos/locked-gold /what-is-celo/about-celo-l1/protocol/pos/locked-gold
+/protocol/pos/penalties /what-is-celo/using-celo/protocol/penalties
+/protocol/pos/validator-elections /what-is-celo/about-celo-l1/protocol/pos/validator-elections
+/protocol/pos/validator-groups /what-is-celo/about-celo-l1/protocol/pos/validator-groups
+/protocol/pos/validator-rewards /what-is-celo/about-celo-l1/protocol/pos/validator-rewards
+/protocol/pos/epoch-rewards /what-is-celo/about-celo-l1/protocol/pos/epoch-rewards
+/protocol/pos/epoch-rewards-validator /what-is-celo/about-celo-l1/protocol/pos/epoch-rewards-validator
+/protocol/pos/epoch-rewards-locked-gold /what-is-celo/about-celo-l1/protocol/pos/epoch-rewards-locked-gold
+/protocol/pos/epoch-rewards-community-fund /what-is-celo/using-celo/protocol/epoch-rewards/community-fund
+/protocol/pos/epoch-rewards-carbon-offsetting-fund /what-is-celo/using-celo/protocol/epoch-rewards/carbon-offsetting-fund
+/protocol/consensus/index /what-is-celo/using-celo/protocol/consensus
+/protocol/consensus/validator-set-differences /what-is-celo/about-celo-l1/protocol/consensus/validator-set-differences
+/protocol/consensus/locating-nodes /what-is-celo/about-celo-l1/protocol/consensus/locating-nodes
+/protocol/consensus/ultralight-sync /what-is-celo/using-celo/protocol/consensus
+/protocol/identity/index https://self.xyz/
+/protocol/randomness /what-is-celo/about-celo-l1/protocol/randomness
+/protocol/contracts/add-contract /what-is-celo/about-celo-l1/protocol/contracts/add-contract
+/protocol/transaction/erc20-transaction-fees /what-is-celo/using-celo/protocol/transaction
```

Comment:
/what-is-celo/using-celo/protocol/transaction doesn't work

Reply:
There is another redirect for the same link below, so I recommend removing the line